### ISSUE TITLE
Missing Data: {externalConnectors.externalConnection-id}

### ISSUE THREAD

**Issue by zengin** [microsoftgraph/msgraph-sdk-raptor#617]
Test Cases:
- [x] #691
- [x] #692
- [x] #693
- [x] #694

**Comment by georgend** [comment]
Odd DevX Behavior for:-

 get-externalconnection-csharp-v1-executes
```http
GET https://graphexplorerapi.azurewebsites.net/permissions?requesturl=/external/connections/raptor&method=GET&scopeType=Application


[
    {
        "value": "ExternalConnection.ReadWrite.OwnedBy",
        "consentDisplayName": "Read and write external connections",
        "consentDescription": "Allows the app to read and write external connections without a signed-in user. The app can only read and write external connections that it is authorized to, or it can create new external connections. ",
        "isAdmin": false
    }
]
```
 list-externalconnection-csharp-v1-executes
```http
GET https://graphexplorerapi.azurewebsites.net/permissions?requesturl=/external/connections&method=GET&scopeType=Application

{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.4",
    "title": "Not Found",
    "status": 404,
    "traceId": "00-1a0e577147b0744591d1109f14a1ec9d-395eedb80174fe4b-00"
}
```

**Comment by fey101** [comment]
3 of the listed issues resolved and 1 known issue of ephemeral data #692 created
